src/main/scala/nl.knaw.dans.easy.curationwork/Report.scala

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.curationwork
+
+import java.nio.file.{Files, Path, Paths}
+
+import sys.process._
+import org.apache.commons.configuration.PropertiesConfiguration
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import resource.managed
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.io.FileUtils
+
+import scala.annotation.tailrec
+import scala.collection.JavaConverters._
+import scala.io.StdIn
+import scala.language.postfixOps
+import scala.util.Try
+import scala.xml.XML
+
+
+class Report(configuration: Configuration) extends DebugEnhancedLogging {
+
+  private def getCurationDirectory(datamanager: Option[DatamanagerId]): Path = {
```

Comment:
duplicate from `Assign` and `Unassign`? Same for other functions...
Suggestion: make a class for functions related to the curation directory. All basic I/O and CRUD operations go there. Then you use this class in these `Report`, etc. classes. That makes these things much more testable.

Perhaps, by doing so, you'll find that the `Report` etc. classes can actually just be 3 functions in one class.

Reply:
I had pushed changes while I was still in the middle of changing the classes structure.  Now I have EasyManageCurationWorkApp class, where these methods are housed, and Assign, Unassign  and Report are its subclasses.